scripts/provision.sh

```diff
@@ -272,23 +272,24 @@ EOT
 }
 
 install_frontend() {
-  if hab svc status habitat/builder-api >/dev/null 2>&1; then
+  #Check if api and datastore services are already configures or installed.
+  if sudo hab svc status habitat/builder-api >/dev/null 2>/dev/null; then
```

Comment:
We don't want to have sudo embedded in a script.  It is better to invoke sudo at the cmd line,

Reply:
Ignore the above - we have been using sudo in this script and have it wrapped in a function.